docs/configuration.html

```diff
@@ -736,8 +316,18 @@
 
 
 <p>More details about consumer configuration can be found in the scala class <code>kafka.consumer.ConsumerConfig</code>.</p>
-<h3><a id="producerconfigs">3.3 Producer Configs</a></h3>
-Essential configuration properties for the producer include:
+
+    <h3><a id="newconsumerconfigs">3.3 New Consumer Configs</a></h3>
+    We are working on a replacement for our existing simple and high-level consumers. The code is available in trunk now and can be considered beta quality. Below is the configuration for the new consumer: </p>
```

Comment:
We can probably mention "Since 0.9.0 release we have been working on ..." and remove "available in trunk".